src/QsCompiler/Tests.Compiler/TestCases/LocalVerification.qs

```diff
@@ -1165,6 +1174,15 @@ namespace Microsoft.Quantum.Testing.LocalVerification {
     @ DeprecatedAttribute()
     newtype UsingDeprecatedAttribute3 = Unit;
 
+    @ Deprecated("")
+    function DeprecatedAttributeInDeprecatedCallable() : Unit {
+        UsingDeprecatedAttribute1();
+    }
+
+    function UsingDepAttrInDepCall() : Unit {
+        DeprecatedAttributeInDeprecatedCallable();
+    }
+
```

Comment:
Could we also add a test for calling a deprecated type's constructor (like `DeprecatedType()`) in a deprecated callable?